packages/keymanager-server/src/impl.ts

```diff
@@ -0,0 +1,226 @@
+import {SecretKey} from "@chainsafe/bls";
+import {Keystore} from "@chainsafe/bls-keystore";
+import {
+  Api,
+  DeletionStatus,
+  ImportStatus,
+  KeystoreStr,
+  SlashingProtectionData,
+} from "@chainsafe/lodestar-api/keymanager";
+import {fromHexString} from "@chainsafe/ssz";
+import {Interchange, ISlashingProtection, SecretKeyInfo, SignerType, Validator} from "@chainsafe/lodestar-validator";
+import {PubkeyHex} from "@chainsafe/lodestar-validator/src/types";
+import {Root} from "@chainsafe/lodestar-types";
+import {unlink, writeFile} from "fs/promises";
+import lockfile from "lockfile";
+import {ILogger} from "@chainsafe/lodestar-utils";
+
+export const LOCK_FILE_EXT = ".lock";
```

Comment:
Ok so this is the situation: Both the cli package and the keymanager package need the lock file extension. It was originally defined in the cli package (in keys.ts) but I moved it to `packages/cli/src/util/lockfile.ts`, with the hope of reusing it across `cli` package and the `keymanager` when I moved the lockfile.ts into the `lodestar-util` - but I have since learnt that `lodestar-util` should be node agnostic.

But for the lock file extension to not be duplicated, it needs to be defined somewhere it can be used from both `cli` and `keymanager` package. Where do you suggest?